sycl/doc/extensions/proposed/sycl_ext_intel_fpga_kernel_interface_properties.asciidoc

```diff
@@ -192,22 +192,31 @@ inline constexpr pipelined_key::value_t<
  `register_map_interface_do_not_wait_for_done_write`.
 
 |`pipelined`
-|An unsigned integer value is accepted as property parameter.
+|A signed integer value is accepted as property parameter.
 
  When the parameter is set to a non zero value, the property directs the
  compiler to pipeline calls to the kernel such that multiple invocations of the
- kernel can be in flight simultaneously. The parameter value also specifies the
- minimum number of cycles between successive invocations of the kernel. Example:
+ kernel can be in flight simultaneously.
+
+ When the parameter is a positive value, the value specifies the 'initiation
+ interval' (II) of the kernel i.e., the minimum number of cycles between successive
+ invocations. Example:
+
+ * `pipelined<N>` - For `N > 0`, the compiler will pipeline multiple kernel
+ invocations such that an invocation can be launched every `N` cycles if one is
+ available.
+
+ When the parameter is set to `-1`, the compiler will determine the II and
+ pipeline the kernel invocations.
 
- * `pipelined<N>` - The compiler will pipeline multiple kernel invocations such
- that an invocation can be launched every `N` cycles if one is available.
- 
  When the parameter is set to `0`, the compiler will not pipeline kernel
  invocations.
 
  If the `pipelined` property is not specified, the default behavior is
  equivalent to a combination of the property parameter values listed above, but
  the choice is implementation defined.
+
+ If the property parameter is not specified, the default value is `-1`.
```

Comment:
What happens if `N < -1`? If it isn't a valid value, could you please add a note somewhere in this section specifying that valid values are `N >= -1`?